moped-database/metadata/tables.yaml

```diff
@@ -3288,56 +3292,66 @@
     - role: moped-admin
       permission:
         columns:
-          - date_added
           - is_current_phase
-          - is_deleted
-          - phase_description
-          - phase_end
+          - is_phase_end_confirmed
+          - is_phase_start_confirmed
+          - created_by_user_id
           - phase_id
-          - phase_start
           - project_id
           - project_phase_id
           - subphase_id
-          - is_phase_start_confirmed
-          - is_phase_end_confirmed
+          - phase_description
+          - created_at
+          - phase_end
+          - phase_start
+          - updated_by_user_id
+          - is_deleted
+          - updated_at
```

Comment:
the column permission changes are:
- remove `date_added`
- add: created_at, updated_at, created_by_user_id, updated_by_user_id
- configure `set` rules for `created_by_user_id` (insert only) and updated_by_user_id (insert and update)